### PR TITLE
footer component

### PR DESCRIPTION
**Pull request by william-man**
Created footer component with links.

### CODE REVIEW

**Comment by william-man** [comment]
Added contributions to footer

File: src/views/mixins/footer.pug [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+        .Nav-Links 
+            a(href="/") Home 
+            a(href="" , rel="noopener noreferrer") Services 
+            a(href="", rel="noopener noreferrer") Blog/News
+        
+        .Github  
+            h4 Contributors
+             a(href="https://github.com/william-man", target="_blank", rel="noopener noreferrer")  William
+            a(href="https://github.com/halimahexe", target='__blank' rel="noopener noreferrer") Halimah
```
Comment (halimahexe) [inline src/views/mixins/footer.pug]: Please can you correct `target='__blank'` to `target='_blank'`?

File: src/views/pages/home/index.pug [hunk 2]
```diff
@@ -11,4 +12,6 @@ block content
         h2.heading.foobar Hello world 
         div#potato.chips
             p Hot Potatoes
-            +test()
\ No newline at end of file
+            +test()
```
Comment (halimahexe) [inline src/views/pages/home/index.pug]: I think we can get rid of this test content - do you want to do it here?

**Review by halimahexe — CHANGES_REQUESTED** [review]
Just left a few comments for you to resolve before merging! :)

File: src/views/pages/home/index.pug (hunk 2 above)
Comment (william-man) [reply src/views/pages/home/index.pug]: Removed test and elements

File: src/views/mixins/footer.pug (hunk 1 above)
Comment (william-man) [reply src/views/mixins/footer.pug]: fixed typo

**Pull request merged**